nemo/collections/asr/modules/rnnt.py

```diff
@@ -245,7 +253,18 @@ def predict(
         del y, start, state
         return g, hid
 
-    def _predict(self, vocab_size, pred_n_hidden, pred_rnn_layers, forget_gate_bias, t_max, norm, dropout):
+    def _predict(
+        self,
+        vocab_size,
+        pred_n_hidden,
+        pred_rnn_layers,
+        forget_gate_bias,
+        t_max,
+        norm,
```

Comment:
Descriptions of the newe params are missing.

Reply:
Good catch